### DISCUSSION TITLE
Bicycle icon in lane images - more copyright paranoia

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4321] [Q&A] [answered]
@westnordost Do you remember source of bicycle image used for example in https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/cycleway/none_no_oneway.svg ?

I admit that it is likely [PD-shape](https://commons.wikimedia.org/wiki/Template:PD-shape) anyway, but if you remember that you made it on your own or remember its source that would make it even easier.

**Comment by westnordost** [comment, marked as answer]
It's the white version of the yellow bicycle from the quest icon. In other words, I made it myself.
